resources/index.html

```diff
@@ -174,11 +174,11 @@ <h2>Chinese Script Overview</h2>
 
 <p>Words are not separated by spaces or any other character. There is no case distinction. The visual forms of characters don't interact.</p>
 
-<p>In its 'main' category, CLDR lists 2,210 characters for the Simplified Chinese orthography, and 2,180 for Traditional Chinese. Combined, this includes 3,026 unique characters, and an overlap of 1,064 characters. A working set of characters for modern Chinese may include 3 times this number, and the Unicode Standard includes approaching 100,000 Han characters, many of which are archaic or esoteric.</p>
+<p>In its 'main' category, CLDR lists 2,210 characters for the Simplified Chinese orthography, and 2,180 for Traditional Chinese. Combined, this includes 3,026 unique characters, and an overlap of 1,064 characters. A working set of characters for modern Chinese may include 3 times this number, and number of characters in the Unicode Standard approaches 100,000 Han code points, many of which are archaic or esoteric. In fact, various regions define their own character sets, such as the 3,500 characters in the Tier I Table of <span lang="zh">通用规范汉字表</span> (General Standard Chinese Characters) in Mainland China, the 4,808 characters in the Taiwanese <span lang="zh">常用“国字”标准字体表</span> (Chart of Standard Forms of Common National Characters), the 4,759 characters in <span lang="zh">常用字字形表</span> (Common Chinese Characters) in Hong Kong SAR, or the sets of <span lang="zh">欢乐伙伴</span> (&quot;Happy Buddy&quot;) characters for Singaporean primary schools.</p>
```

Comment:
The task force prefers to remove the CLDR numbers unless they have reliable sources.

See discussions in https://www.w3.org/2024/05/08-clreq-minutes.html#t01

Reply:
Ok. I commented that text out because i wanted the information to still be available to myself for future discussions re. CLDR.  The character sets now mentioned are all fairly small, relatively speaking, so i added the figure of 10,000 to give the impression of the size of the repertoire needed for things such as text editors (I know that Mainland China basic repertoire is substantially less than the Taiwanese, and this is only an indicator.)